sqlstore/src/main/resources/db/migration_beforeMigrate/05_standard.sql

```diff
@@ -419,6 +419,32 @@ CREATE PROCEDURE addTissueType(
   END IF;
 END//
 
+--  Use inconjunction with the spreadsheet formula: =CONCATENATE("CALL addIdentity('",A2,"', '",B2,"', '",C2,"', ", IF(ISBLANK(D2), "NULL", CONCATENATE("'", D2, "'")), ", ", IF(ISBLANK(E2), "NULL", CONCATENATE("DATE '", E2, "'")), ", '", F2, "', '", G2, "');")
+DROP PROCEDURE IF EXISTS addIdentity//
+CREATE PROCEDURE addIdentity(
+  iProject varchar(255),
+  iExternalName varchar(255),
+  iAlias varchar(255),
+  iDescription varchar(255),
+  iDate date,
+  iScientificName varchar(255),
+  iSampleType varchar(255)
+) BEGIN
+  DECLARE newSampleId bigint(20);
+  
+  IF NOT EXISTS (SELECT 1 FROM Identity WHERE externalName = iExternalName) THEN
```

Comment:
We should probably also have an "else" case that throws an error to indicate that an identity with that external name already exists in the project. Right now it returns `Query OK, 0 rows affected`, which does not clearly indicate that no record was created, or why.